qbertclient/keystone.py

```diff
@@ -88,22 +88,23 @@ def get_token(self):
         resp = requests.post(url,
                              data=json.dumps(body),
                              headers={'content-type': 'application/json'},
-                             verify=self.verify)
+                             **kwargs)
```

Comment:
In the old version of the code, how was `self.verify` initialized? If it is a property of the client, then it seems  simpler for the user to specify `verify` once at client initialization time, and then all methods making requests can send down `verify=self.verify` instead of requiring the user to pass it explicitly on every method. Thoughts?

Reply:
Yes. That may be easier to use at the client level.